CHANGELOG.md

```diff
@@ -15,7 +15,7 @@
 #### New interpreters
 
 * Add support for TruffleRuby 20.1.0, 20.2.0
-* Add support for Ruby 2.4.10, 2.5.8, 2.6.6, 2.7.1 [\#4899](https://github.com/rvm/rvm/pull/4899)
+* Add support for Ruby 2.4.10, 2.5.8, 2.6.6, 2.7.1 [\#4899](https://github.com/rvm/rvm/pull/4899) and 2.7.2 [\#4988](https://github.com/rvm/rvm/pull/4988)
```

Comment:
I just moved the changelog entry to the right place. It looks like 3.0.0-preview1 was added in the wrong place but we can fix that in a separate commit.

Reply:
Oops, that was me for 3.0.0-preview1

Reply:
Oops, that was me for 3.0.0-preview1